### PR TITLE
Update Savers

### PR DESCRIPTION
**Pull request by nschulux**
Last words wrong directory.

Readd Maze , like Rainer suguested.

### CODE REVIEW

**Comment by rabe-soft** [comment]
I think, rename the program to avoid the options is a BAD strategy. We have to find out th reason, not to kill the Symptoms only.

**Review by nschulux — COMMENTED** [review]
Maze readded , like Rainer suguested.
He will check the code for Maze Options.

File: Tools/build/product/bbxensem/Trans/german/MAZE_(LA.000
Comment (nschulux) [inline Tools/build/product/bbxensem/Trans/german/MAZE_(LA.000:1]: Redded Maze from previous PR.

**Review by nschulux — COMMENTED** [review]
Redded Maze from previous PR.

**Comment by nschulux** [comment]
Remake PR.

**Pull request closed without merging**